src/overlays/actors/ovl_Dm_Stk/z_dm_stk.c

```diff
@@ -315,19 +324,14 @@ void DmStk_LoadObjectForAnimation(DmStk* this, PlayState* play) {
     }
 }
 
-/**
- * This function is always called with unusedExtraOffset = 0.
- */
-void DmStk_ChangeAnim(DmStk* this, PlayState* play, SkelAnime* skelAnime, AnimationInfo* animationInfo,
-                      u16 unusedExtraOffset) {
+void DmStk_ChangeAnim(DmStk* this, PlayState* play, SkelAnime* skelAnime, AnimationInfo* animInfo, u16 animIndex) {
     DmStk_LoadObjectForAnimation(this, play);
 
-    animationInfo += unusedExtraOffset;
+    animInfo += animIndex;
 
-    Animation_Change(skelAnime, animationInfo->animation, animationInfo->playSpeed, animationInfo->startFrame,
-                     (animationInfo->frameCount < 0.0f) ? Animation_GetLastFrame(animationInfo->animation)
-                                                        : animationInfo->frameCount,
-                     animationInfo->mode, animationInfo->morphFrames);
+    Animation_Change(skelAnime, animInfo->animation, animInfo->playSpeed, animInfo->startFrame,
+                     (animInfo->frameCount < 0.0f) ? Animation_GetLastFrame(animInfo->animation) : animInfo->frameCount,
```

Comment:
Hmm, interesting. Here we have a ternary without the use of the `endFrame` temp. I tried DmChar08 and it also works as a ternary. Would we want to change everything to use a ternary in the function call itself like here (assuming it works everywhere), keep the temp, or possibly not care?

Reply:
Oh actually yes I prefer the temp here. I remember using a temp in my other PR somewhere, I'll make the change here and in other places